### PR TITLE
Initial stab at switching out logos

### LINKED ISSUE #139: Make header logo switch on subdomain
Currently, the header logo always shows the Baja Pet Rescue logo regardless of which subdomain the visitor is on. Let's make it change to a different logo with a different subdomain.

### PR DESCRIPTION
**Pull request by Thrillberg**
### Describe your change in plain English.
**I do not necessarily endorse my solution here** but I figured pushing up some not-so-great code might be a good conversation-starter! 😆 Anyway, the goal here is to get the logos to switch per the subdomain, which works with this PR. Observe!

Screenshot 2023-08-01 at 9 49 55 PM

Not sure why I called this `alta`. I guess it's just a play on `baja`. But it's probably more confusing than it's worth!

Screenshot 2023-08-01 at 9 50 31 PM

Hey look, Sean has his own pet shelter!

Ok so that's cool but I'm not happy about how I implemented it. I need ideas from you all to make this better! As you can see in the code I've got this `get_organization_settings` hash that will undoubtedly grow to an obnoxious degree. How should we store the settings of our organizations? Should we start mapping that out now? Is it too early? (never!)

It's a bit buried in the diff because I decided to move the header out to its own partial but the organization settings get invoked like this:
```ruby
<%= link_to image_tag(@organization_settings[:logo], class: 'logo me-2', alt: 'site logo'), root_path %>
```

Anyway, what do you think?

### Link to the issue
Fixes #139

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+  def get_organization_settings
+    {
+      "alta" => {
+        logo: "logo.png"
+      },
+      "rubyforgood" => {
+        logo: "rubyforgood-logo.png"
+      }
+    }
+  end
```
Comment (edwinthinks) [inline app/controllers/application_controller.rb:27]: Hmm maybe we could instead use a attachment on Organization with ActiveStorage. So like `organization.logo`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -16,121 +16,7 @@
...
-                    data: { turbo: false },
-                    method: 'delete' %>
-              </li>
-            <% end %>
-          </ul>
-        </div>
-      </div>
-    </nav>
-
+    <%= render "layouts/header" %>
```
Comment (edwinthinks) [inline app/views/layouts/application.html.erb:19]: I would either pass in organization or create a helper called `current_organization` to fetch the organization to configure the page views.

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<nav class="navbar navbar-expand-lg navbar-dark"
+      style='background-color: #00401F;' aria-label="Fourth navbar example">
+  <div class="container">
+    <div class="navbar-brand">
+      <%= link_to image_tag(@organization_settings[:logo], class: 'logo me-2', alt: 'site logo'),
```
Comment (edwinthinks) [inline app/views/layouts/_header.html.erb:5]:
```suggestion
      <%= link_to image_tag(Current.organziation.logo, class: 'logo me-2', alt: 'site logo'),
```
An idea of how to pass around the organization to make tweaks on the UI.

**Review by edwinthinks — COMMENTED** [review]
Thanks for starting this convo starter! I added some more commentary on the approach in Slack.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (edwinthinks) [reply app/controllers/application_controller.rb:27]: Technically I think we could use rich text columns for the various pages on the Organization to get that wysiwyg experience. It won't be super super customizable but enough to maybe be different enough?

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (nsiwnf) [reply app/views/layouts/_header.html.erb:5]: or `current_tenant.logo` ?
